pkg/sql/exec/distinct.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2018 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package exec
+
+// sortedDistinctInt64Op runs a distinct on the column in sortedDistinctCol,
+// writing the result to the bool column in outputColIdx by or'ing the
```

Comment:
[nit] This comment needs some rephrasing (it sounds as if it's doing some bitwise OR stuff). I'd just say that the bool column is set to true for every value that differs from the previous one.